docs/001_develop/03_client-capabilities/008_interaction/modal.mdx

```diff
@@ -179,7 +179,7 @@ The sections below provide property and attribute binding examples for Genesis C
         <tr>
             <td>type</td>
             <td><code>string</code></td>
-            <td>The type of dialog on of <code>default</code>, <code>alert</code> or <code>success</code>. <strong>Defaults to <code>default</code></strong>. This changes the styling depending on value.</td>
+            <td>The type of dialog one of <code>default</code>, <code>alert</code> or <code>success</code>. <strong>Defaults to <code>default</code></strong>. This changes the styling depending on value.</td>
```

Comment:
Check this locally because I am not sure the syntax is correct. Also if we can give any info about how the styling changes, that would be great
```suggestion
           <td>code>default</code> \| <code>alert</code> \| <code>success</code></td>
            <td>The type of dialog. <strong>Defaults to <code>default</code></strong>. This changes the styling depending on value.</td>
```